cranelift/codegen/src/ir/immediates.rs

```diff
@@ -1470,7 +1483,7 @@ mod tests {
     fn fcvt_to_sint_negative_overflow_ieee32() {
         for n in &[8, 16] {
             assert_eq!(-((1u32 << (n - 1)) as f32) - 1.0, unsafe {
-                mem::transmute(Ieee32::fcvt_to_sint_negative_overflow(*n))
+                mem::transmute::<_, f32>(Ieee32::fcvt_to_sint_negative_overflow(*n))
```

Comment:
Not sure why this was needed but `rustc` started giving me failed-to-infer-types errors for this all of a sudden.

Reply:
I've seen those come up when unrelated bits fail to typecheck; perplexing to me as well.

Now that things typecheck, though, could you try removing this?